src/Inkluzitron/Models/BdsmTraitOperationCheck.cs

```diff
@@ -1,22 +1,22 @@
 ﻿using Discord;
+using Inkluzitron.Data;
 using Inkluzitron.Enums;
 using Inkluzitron.Extensions;
 using Inkluzitron.Models.Settings;
-using Inkluzitron.Services;
 using System;
 
 namespace Inkluzitron.Models
 {
     public class BdsmTraitOperationCheck
     {
         private readonly BdsmTraitOperationCheckTranslations _translations;
-        private readonly UsersService _usersService;
+        private readonly BotDatabaseContext _dbContext;
```

Comment:
Prosim ne.
Ten údaj ať zjistí ten, kdo vytváří instanci téhle třídy. Tohle je jenom datová struktura, složitý věci jako DB kontext by tu být neměly.

Reply:
Paráda, díky.